asapdiscovery-alchemy/asapdiscovery/alchemy/cli/utils.py

```diff
@@ -2,8 +2,26 @@
 
 
 def print_header(console: "rich.Console"):
-    "Print an ASAP-Alchemy header"
+    """Print an ASAP-Alchemy header"""
 
     console.line()
     console.rule("ASAP-Alchemy")
     console.line()
+
+
+def pull_from_postera(molecule_set_name: str):
```

Comment:
perhaps this should be defined in `asapdiscovery-data/postera` instead?

Reply:
I would like to keep it separate if possible as it allows us to tuck the imports so you don't end up importing a load of redundant packages if you don't want to use postera.